IntegrationTests/test_runner.py

```diff
@@ -0,0 +1,352 @@
+"""
+This is the SOM integration test runner file. Pytest automatically discovers
+this file and will find all .som test files in the below directories.
+"""
+
+import subprocess
+from pathlib import Path
+import os
+import sys
+import pytest
+import yaml
+import conftest as external_vars
+
+
+def debug(message):
+    """
+    Take a string as a mesasage and output if DEBUG is true
+    """
+    if external_vars.DEBUG is True:
+        print(message)
+
+
+def debug_list(message_list, prefix="", postfix=""):
```

Comment:
do we really need these debug methods? I would rather avoid them. `DEBUG` itself is something I'd probably avoid.